presto-native-execution/presto_cpp/main/TaskManager.cpp

```diff
@@ -238,6 +253,7 @@ std::unique_ptr<TaskInfo> TaskManager::createOrUpdateErrorTask(
 }
 
 void TaskManager::getDataForResultRequests(
+    const std::shared_ptr<PrestoTask> task,
```

Comment:
Should we pass const reference here to avoid shared pointer copy?

Reply:
yes. Thanks!